pkg/views/workspace/selection/provider.go

```diff
@@ -17,12 +17,24 @@ import (
 
 var titleStyle = lipgloss.NewStyle()
 
-func selectProviderPrompt(gitProviders []gitprovider_view.GitProviderView, additionalProjectOrder int, choiceChan chan<- string) {
+func selectProviderPrompt(gitProviders []gitprovider_view.GitProviderView, additionalProjectOrder int, selectedReposGitProvider map[string]bool, choiceChan chan<- string) {
```

Comment:
I can understand why you implemented this check but I think covering this edge case is going a bit too far. Moreover, I believe you only covered the case where the user is not part of any namespace (organization, project, etc.). 
In my case, after I select a provider (Github for example), I first select an organization and then I select a repo from the list. This means that this check would have to go one step further and disable organizations that have all of their repos selected.

I believe the code and logic complications that this implies doesn't make it worth it to cover such an edge case.

@idagelic do you agree?

Reply:
@Tpuljak  hey i would say lets keep it, say a scenario where user has entered some 8-9 github provider repos already, and he/she again selects github provider and finds all repos to be selected, In this case there is no way of returning and he ends up getting to enter all repos from the start..

I would look and implement for this organizations logic too...

Reply:
I understand but okay. If you're wiling to implement the logic for organizations as well, you can do so.

Reply:
@Tpuljak hey I have created 2 orgs from my github account, but they dont get listed in the multi-project command..

Reply:
@abhishek818 that might be because of #731.

You can check out the easy fix here https://github.com/daytonaio/daytona/pull/731/files and apply it to your local branch but don't push it to this PR.

Reply:
thanks for the quick response man.. i was putting logs here there and midway of wasting some long time.

Reply:
fixed..